src/os/sockaddr.php

```diff
@@ -0,0 +1,33 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\OS;
+
+use namespace HH\Lib\_Private\_OS;
+
+/** Address of a socket.
+ *
+ * @see `sockaddr_un` for UNIX domain sockets.
+ * @see `sockaddr_in` for INET (IPv4) sockets.
+ * @see `sockaddr_in6` for INET6 (IPv6) sockets.
+ */
+abstract class sockaddr {
```

Comment:
```suggestion
abstract class SocketAddress {
```

using lowercase for type names in Hack just seems weird and completely inconsistent, I'm aware that this follows C and is done to ease discoverability, but I think the documentation should point people the right way and have the type names in hack be consistent